keyboards/skeletonkbd/frost68/frost68.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2023 skeletonkbd (@skeletonkbd)
```

Comment:
Empty file, can be removed

Reply:
Deleted frost68.h
5ffd1a108140c195c87d65d923d2bb5759bb66e5